app.rb

```diff
@@ -0,0 +1,164 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+# Menu blueprint
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/MethodLength
+class Menu
+  def initialize
+    @user_choice = 0
+  end
+
+  def menu
+    puts "\nPlease choose an option by entering a number:\n
+            \n1 - List all books
+            \n2 - List all people
+            \n3 - Create a person\n4 - Create a book\n5 - Create a rental\n6 - List all rentals for given id\n7 - Exit"
```

Comment:
[**OPTIONAL**] Try this on your console, you wouldn't need to use escape character operator `\` and it will look like what the client (project requirements :wink:) wanted :+1: 
```suggestion
    puts "Please choose an option by entering a number:
1 - List all books
2 - List all people
3 - Create a person
4 - Create a book
5 - Create a rental
6 - List all rentals for given id
7 - Exit"
```